pyproject.toml

```
@@ -325,6 +325,8 @@ extend-ignore-re = [
   "lightning",
   # `typos` flags 'lok' as a typo of 'look'
   "(?i)daniel lok",
+  # name of committer
+  "@4n4nd",
```

```suggestion
  # GitHub handle
  "@\\w+",
```

not sure if `\\w+` works. let's test it.

---

looks like it works!